data/json/mutations/mutations.json

```
@@ -5039,7 +5039,7 @@
     "points": -2,
     "visibility": 10,
     "ugliness": 10,
-    "description": "Your flesh is a pleasing gel-like consistency.  Your bodily functions seem to be moving around, and your leg-equivalents flow comfortably.",
+    "description": "Your flesh is a pleasing gel-like consistency.  Your bodily functions seem to be moving around, but your leg-equivalents flow slower.",
```

```suggestion
    "description": "Your flesh is a pleasing gel-like consistency.  Your bodily functions seem to be moving around, and your leg-equivalents flow comfortably - if a little slower than your old meat-legs.",
```

In your edit it's unclear what they flow slower than.

---

Solid edit. You word it masterfully, impressive, I spent some time thinking of a good way to word it but yours is great.